Assets/Scripts/UI/UIHealthSetter.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using UnityEngine;
+
+public class UIHealthSetter : MonoBehaviour {
+	private UIValueBar bar;
+	private Health playerHealth;
+
+	private void Start() {
+		bar = GetComponent<UIValueBar>();
+		GameObject go = GameObject.FindWithTag("Player");
+		if (go != null)
+			playerHealth = go.GetComponent<Health>();
+
+		if (playerHealth != null) {
+			bar.MaxValue = playerHealth.MaxValue;
+			bar.Value = playerHealth.Value;
+		}
+	}
+
+	private void Update() {
```

Comment:
Would it be fitting to use lateupdate here to really get the player's hp?

Reply:
Good idea!